app/i18n/locales/pt-BR.json

```diff
@@ -757,5 +757,6 @@
   "The_user_wont_be_able_to_type_in_roomName": "O usuário não poderá digitar em {{roomName}}",
   "The_user_will_be_able_to_type_in_roomName": "O usuário poderá digitar em {{roomName}}",
   "Enable_writing_in_room": "Permitir escrita na sala",
-  "Disable_writing_in_room": "Desabilitar escrita na sala"
+  "Disable_writing_in_room": "Desabilitar escrita na sala",
+  "You_dont_have_permission": "Você não tem permissão para realizar esta ação. Verifique com o proprietário da sala."
```

Comment:
`You_dont_have_permission` for what?
This label is "misleading", there is no problem with placing large labels, as long as they bring coherent meanings to the context.
Maybe: `You_dont_have_permission_to_perform_this_action`